components/webfield/AreaChairConsole.js

```diff
@@ -357,6 +365,22 @@ const AreaChairConsole = ({ appContext }) => {
         : Promise.resolve([])
       // #endregion
 
+      // #region secondary ACs
+      const secondaryAreaChairsP = secondaryAreaChairName
+        ? api
+            .get(
+              '/edges',
+              {
+                invitation: `${venueId}/${secondaryAreaChairName}/-/Assignment`,
```

Comment:
is it possible to pass the whole invitation id in the property of the component?

Reply:
@xkopenreview I left a comment here